src/main/java/weavers/siltarae/global/util/CryptoUtil.java

```diff
@@ -39,15 +43,24 @@ public static String encrypt(String value) {
 
     public static String decrypt(String encrypted) {
         try {
-            byte[] keyBytes = secretKey.getBytes(StandardCharsets.UTF_8);
-            Key key = new SecretKeySpec(keyBytes, ALGORITHM);
-            Cipher cipher = Cipher.getInstance(CIPHER_TRANSFORMATION);
-            cipher.init(Cipher.DECRYPT_MODE, key);
+            Cipher cipher = createCipherForMode(Cipher.DECRYPT_MODE);
 
             byte[] original = cipher.doFinal(Base64.getDecoder().decode(encrypted));
             return new String(original, StandardCharsets.UTF_8);
         } catch (Exception e) {
             throw new BadRequestException(ExceptionCode.FAIL_DECRYPT);
         }
     }
+
+    private static Cipher createCipherForMode(int encryptMode) throws NoSuchAlgorithmException, NoSuchPaddingException, InvalidKeyException, InvalidAlgorithmParameterException {
```

Comment:
```suggestion
    private static Cipher createCipherForMode(int encryptMode) throws Exception {
```
메서드 분리해주신거 좋아요! 확실히 코드의 중복이 줄어든 것 같아요 👏 
외부에서 Exception으로 퉁쳐서 받고 있는데 이런 형태는 어떨까요 👀